tests/agent_run_spec.rb

```diff
@@ -63,6 +63,6 @@
       .should be_file
     file('/var/opt/lib/pe-puppet/state/resources.txt')
       .content
-      .should match /notify\[Hello Puppet!\]/
+      .should match /notify\[hello puppet!\]/
```

Comment:
`resources.txt` file format apparently changed and resource titles are all downcased now. Raised the issue with Henrik but unclear when/why it was changed. This is a simple fix to get things working again.